Jenkinsfile_CNP

```diff
@@ -74,6 +86,28 @@ withPipeline("nodejs", product, component) {
             env.IDAM_OAUTH2_CLIENT_ID='sscs'
     }
 
+     afterSuccess('smoketest:preview') {
```

Comment:
@gokul-sol  why are we introducing playwright tests in preview when we know they are not stable consistently ? @sanjayparekh-hmcts  you aware of this ?

Reply:
@nitinprabhuhmcts this is a placeholder step which enables us to check if tests are running successfully against AAT in Jenkins before pushing it to Master. We will remove this step before pushing the code.